### DISCUSSION TITLE
Upgraded from CC2531 to Electrolama zig-a-zig-ah! (zzh!)

### DISCUSSION THREAD

**Discussion by bartplessers** [Koenkk/zigbee2mqtt#5259] [General]
I recently did a hardware upgrade from a CC2531 to Electrolama zig-a-zig-ah! (zzh!) usb stick.
What I did:
- stop zigbee2mqtt
- changed my configuration:
  - serial --> ttyUSB0
  - pan_id: 6758
- plugged in my Electrolama stick
- rebooted system

However,
- I had to repair every device again
- for some reason I had to change a lot of the batteries of my XIAOMI/IKEA devices. May because my "old" zigbee network was still in the air without coordinator? No clue but I had to renew the battery in +/- 60% of my devices!
- paring was sometimes more difficult compared to my CC2531 stick.

This was rather an unpleasant experience. 
Definitely because I had to go all over my place and find the (sometimes hard to find) buttons to reconnect etc..

So my question(s):
- was there another way to upgrade my hardware without the need of repairing all devices?
- can the battery loss be explained? Or was this just "bad luck"
- is there a way to have a "disaster" setup: another RPI running my old CC2531 stick, that can be promoted to coordinator of the same zigbee network without overdoing the whole thing again?

kind regards,
Bart

**Comment by Koenkk** [comment]
> was there another way to upgrade my hardware without the need of repairing all devices?

No, repairing is required in this situation (https://www.zigbee2mqtt.io/information/FAQ.html#requires-repairing)

> can the battery loss be explained? Or was this just "bad luck"

Why did you renew them? If they did not pair it may indeed by because your old network was still running. It's recommended to shut it off entirely by removing power from every router

> is there a way to have a "disaster" setup: another RPI running my old CC2531 stick, that can be promoted to coordinator of the same zigbee network without overdoing the whole thing again?

No

**Reply by bartplessers** [reply]
>Why did you renew them? 
they run out of power. A lot of devices suddenly were not responsive due to lack of power :-(

anyway. Everything up-and-running again. Thanx for response.

regards,
B

**Comment by JanHACS** [comment]
Hi @bartplessers, I am going to change from Conbee II to a electrolama that I have recently ordered, because I have some range problems with my garage devices. Did you notice a better zigbee network range?

**Reply by Snakesms** [reply]
I've observed stability improvement and some new devices become available that didn't previously work with CC2531 but network range (without routers) become even  worse

**Reply by DrTron** [reply]
I experienced the same when I switched to a Slaesh CC2652RB stick.
Get a CC2530+CC2591, a small 3.3V power supply and a box. Works wonders.
https://www.zigbee2mqtt.io/how_tos/how_to_create_a_cc2530_router.html

**Comment by bartplessers** [comment]
@JanHACS 
not really.
"maybe" there is a bigger range with the electrolama: since I was using some Eurotronic thermostatic valves (https://www.zigbee2mqtt.io/devices/SPZB0001.html) and one of them gave some problems. After I used electrolama, the valve worked correctly, but I already ordered a new valve (moes https://www.zigbee2mqtt.io/devices/TS0601_thermostat.html) and also replaced the problematic valve. So I'm not sure what caused the problem and what is the solution.

However, I like the electrolama because no special hardware needed for (re)flashing. I was a l little bit disappointed in the sending costs (15EUR for a device of 30EUR...), but no regrets of buying it.

kind regards,
B
